From a16c56581a088f279a0f482fb5e78364c50204c9 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:43:58 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(ca17c7e)..lineage-16.0-android-9.0.0_r31(ed0cf3d) (DO NOT
 SUBMIT)

Bill Yi (1):
 - Import translations. DO NOT MERGE

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (1):
 - Snap for 5073521 from ec23b2789765e0d9535692b4d5a3c2f01f821df5 to
   pi-qpr2-release

Change-Id: I21442c3217c9fd710663dfd25a85540852a45449

Change-Id: I2cf195cefacc4fdc6ed8cdacc31f51c63cb04ced
---
 res/values-de/strings.xml | 2 +-
 res/values-hr/strings.xml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/values-de/strings.xml b/res/values-de/strings.xml
index 5f5169d..fbaf25d 100644
--- a/res/values-de/strings.xml
+++ b/res/values-de/strings.xml
@@ -49,7 +49,7 @@
     <string name="find_wifi_direct" msgid="5270504288829123954">"Wi-Fi Direct-Drucker suchen"</string>
     <string name="wifi_direct_printers" msgid="541168032444693191">"Wi-Fi Direct-Drucker"</string>
     <string name="searching" msgid="2114018057619514587">"Wird gesucht…"</string>
-    <string name="connect_hint_text" msgid="587112503851044234">"Möglicherweise musst du diese Verbindung auf dem Display deines Druckers bestätigen"</string>
+    <string name="connect_hint_text" msgid="587112503851044234">"Eventuell musst du diese Verbindung auf dem Display deines Druckers bestätigen"</string>
     <string name="connecting_to" msgid="2665161014972086194">"Verbindung zu <xliff:g id="PRINTER">%1$s</xliff:g> wird hergestellt"</string>
     <string name="failed_printer_connection" msgid="4196305972749960362">"Es konnte keine Verbindung zum Drucker hergestellt werden"</string>
     <string name="failed_connection" msgid="8068661997318286575">"Es konnte keine Verbindung zu <xliff:g id="PRINTER">%1$s</xliff:g> hergestellt werden"</string>
diff --git a/res/values-hr/strings.xml b/res/values-hr/strings.xml
index 665e083..98d4507 100644
--- a/res/values-hr/strings.xml
+++ b/res/values-hr/strings.xml
@@ -55,6 +55,6 @@
     <string name="failed_connection" msgid="8068661997318286575">"Povezivanje s pisačem <xliff:g id="PRINTER">%1$s</xliff:g> nije uspjelo"</string>
     <string name="saved_printers" msgid="4567534965213125526">"Spremljeni pisači"</string>
     <string name="forget" msgid="892068061425802502">"Zaboravi"</string>
-    <string name="connects_via_wifi_direct" msgid="652300632780158437">"Povezuje se putem Izravnog Wi-Fi-ja"</string>
+    <string name="connects_via_wifi_direct" msgid="652300632780158437">"Povezuje se putem Izravnog Wi-Fija"</string>
     <string name="connects_via_network" msgid="5990041581556733898">"Povezuje se putem trenutačne mreže na IP adresi <xliff:g id="IP_ADDRESS">%1$s</xliff:g>"</string>
 </resources>
-- 
2.17.1

